packages/experimental/Tooltip/src/Tooltip.types.ts

```diff
@@ -0,0 +1,63 @@
+import type * as React from 'react';
+
+export const tooltip = 'Tooltip';
+
+/**
+ * Properties and Tokens for FluentUI React Native Tooltip
+ */
+export type DirectionalHint =
```

Comment:
>DirectionalHint [](http://example.com/codeflow?start=12&length=15)

Is this shared with Callout or is this different?

Reply:
Shared with Callout so changed to import from there